docs/en/sql-reference/aggregate-functions/reference/exponentialtimedecayedavg.md

```diff
@@ -0,0 +1,105 @@
+---
+slug: /en/sql-reference/aggregate-functions/reference/exponentialtimedecayedavg
+sidebar_position: 108
```

Comment:
We better increase the sidebar position to 109 so that `exponentialTimeDecayedAvg` appears below `exponentialMovingAverage`.

Reply:
@rschu1ze the ordering of sidebar positions in general was a bit of a mess. I have gone through and changed all of the positions so that the sidebar is sorted alphabetically.

Reply:
off topic but I notice that we have functions like `any_respect_nulls`, `anyLast_respect_nulls`, `approx_top_k` which seem to deviate from the camelcase convention of the majority of the functions. Were these just added a long time ago before the convention stuck or is there some reason for it?

Reply:
Thanks for fixing the sidebar positions, that definitely makes sense.

Re function name conventions: Can't really tell, that must have been before my time :-/ Anyways, simply renaming the functions will break existing usage, we can't do that. But adding aliases is totally fine. We might even add an alias and frame the official docs such that the alias is mentioned as the primary name, respectively the original name as the alias 😉 (in any case, let's do such renamings in a separate PR)